### ISSUE TITLE
Plugin causing unnecessary octoprint server reloads

### ISSUE THREAD

**Issue by derocheam** [jneilliii/OctoPrint-Tasmota#60]
Octoprint always asking to reload to update server. 

"Please reload
There is a new version of the server active now, a reload of the user interface is needed. This will not interrupt any print jobs you might have ongoing. Please reload the web interface now by clicking the button below."

I believe this is because the Power data is saved in a label in the octoprint config.yaml. AFAIK when the config.yaml gets updated it requires a reload of octoprint.

  tasmota:
    _config_version: 7
    arrSmartplugs:
    - autoConnect: true
      autoConnectDelay: 10
      autoDisconnect: true
      autoDisconnectDelay: 0
      automaticShutdownEnabled: true
      backlog_off_delay: 0
      backlog_on_delay: 0
      currentState: 'on'
      displayWarning: true
      gcodeEnabled: false
      gcodeOffDelay: 0
      gcodeOnDelay: 0
      icon: icon-bolt
      idx: '1'
      ip: 192.168.1.167
      label: ''
      **label_extended: '<br>ApparentPower: 187.342<br>Power: 83.2<br>Yesterday: 0.785<br>TotalStartTime:
        2020-01-03T00:00:00<br>Current: 1.556<br>Voltage: 120.4<br>Factor: 0.44<br>ReactivePower:
        167.9<br>Total: 84.488<br>Today: 1.271'**

**Comment by jneilliii** [comment]
Thanks for the report. That label by no means was intended to saved into config.yaml. I'll investigate.

**Comment by fionnb** [comment]
I am getting the constand reload requests, too

**Comment by jneilliii** [comment]
@fionnb do you also have a sensor or power monitoring device? Can you guys let me know when you see the reload dialog? You say always, but is it while you keep the interface open or when you first open the page?

**Comment by derocheam** [comment]
It's while I keep the interface open. Happens every couple minutes.

**Comment by jneilliii** [comment]
Thanks, is it equivalent to the polling interval you've set or during printing only? I'm just thinking of the different times I might be checking the status and see if there is a correlation.

**Comment by jneilliii** [comment]
I think I have this issue resolved in the latest 0.8.16rc3 version. If you want to test it out you can copy/paste the URL below into Plugin Manager > Get More > ...from URL and clicking Install.

```
https://github.com/jneilliii/OctoPrint-Tasmota/archive/0.8.16rc3.zip
```
